### PR TITLE
show k8s workers ips after deployment with cli

### PR DESCRIPTION
**Pull request by Eslam-Nawara**
### Description

- show workers ygg ips after deployment with grid-cli

### Changes

- show workers ips if they are set to have a ygg ips

### Related Issues

- #462

### CODE REVIEW

**Comment by xmonader** [comment]
I think workers can have public IPs as well right? and IPv6 too?
And a wireguard one xD

**Comment by Eslam-Nawara** [comment]
> I think workers can have public IPs as well right? and IPv6 too? And a wireguard one xD

of course they can, but the issue only specified ygg ip, fa should public ips also be shown?

**Comment by xmonader** [comment]
> > I think workers can have public IPs as well right? and IPv6 too? And a wireguard one xD
> 
> of course they can, but the issue only specified ygg ip, fa should public ips also be shown?

Yup, please do

**Comment by xmonader** [comment]
I think we are still missing the wireguard IPs 🙈

**Comment by Eslam-Nawara** [comment]
> I think we are still missing the wireguard IPs 🙈

I can't find it actually :)

**Comment by rawdaGastan** [comment]
@Eslam-Nawara you can find it as `IP` in the k8s node struct (the IP is the private wireguard IP)
- https://github.com/threefoldtech/tfgrid-sdk-go/blob/development/grid-client/workloads/k8s.go#L31

File: grid-cli/cmd/deploy_kubernetes.go [hunk 1]
```diff
@@ -196,6 +198,25 @@ var deployKubernetesCmd = &cobra.Command{
 		if ygg {
 			log.Info().Msgf("master yggdrasil ip: %s", cluster.Master.YggIP)
 		}
+
+		for i, worker := range cluster.Workers {
+			log.Info().Msgf("worker%d wireguard ip: %s", i, worker.IP)
```
Comment (rawdaGastan) [inline grid-cli/cmd/deploy_kubernetes.go]: we can ignore `i` and use worker name

**Comment by xmonader** [comment]
Still one more ip (ipv6) of the master we should all of them like we did for the workers

**Comment by Eslam-Nawara** [comment]
It exists here 
- https://github.com/threefoldtech/tfgrid-sdk-go/pull/471/files#diff-6f37cc47e9c5a93162368b6c661fd85de13526e61554eea42aee73f887a343ecR195

**Review by xmonader — APPROVED** [review]

**Pull request merged**